ui/ts/models/log.ts

```diff
@@ -46,17 +48,14 @@ module Models {
         return "Local";
       };
 
-      private _data: Utils.QueryCache<Proto.LogEntry[]> = new Utils.QueryCache((): Promise<Proto.LogEntry[]> => {
-        return m.request({ url: this._url(), method: "GET", extract: nonJsonErrors })
-          .then((results: LogResponseSet) => {
-            return results.d;
-          });
-      });
+      private _data: Utils.QueryCache<Proto.LogEntry[]>;
 
       private _url(): string {
         let url: string = "/_status/logs/";
         if (this.node() != null) {
           url += this.node();
+        } else {
+          url += "local";
```

Comment:
`git reset head^ --patch -- ui/ts/models/log.ts`; select this bit; `git commit --amend`; `git add ui/ts/models/log.ts`; `git commit -m 'tiny bug fix'`